apps/challenges/aws_utils.py

```diff
@@ -1061,6 +1061,334 @@ def scale_resources(challenge, worker_cpu_cores, worker_memory):
         return e.response
 
 
+@app.task
+def detach_policies_and_delete_role(challenge):
```

Comment:
Call this method `delete_eks_infra`. In the method docs just mention that this method detaches policies and deletes roles, and calls delete EFS. Repeat the same with other.